src/components/MovieList.jsx

```diff
@@ -0,0 +1,92 @@
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import "../components/movielist.css";
+
+const MovieList = () => {
+	const [movieList, setMovieList] = useState([]);
```

Comment:
Wonky indentation makes the code harder to read and the fact that it's not consistent (the App.jsx looked good) makes the code look less clean